src/wad/tex.rs

```diff
@@ -423,15 +441,16 @@ fn search_for_frame<'a>(search_for: &WadName, animations: &'a [Vec<WadName>])
 
 
 fn read_sprites(wad: &Archive, textures: &mut BTreeMap<WadName, Image>) -> Result<usize> {
-    let start_index =
-        try!(wad.required_named_lump_index(b"S_START\0")) + 1;
+    let start_index = try!(wad.required_named_lump_index(b"S_START\0")) + 1;
     let end_index = try!(wad.required_named_lump_index(b"S_END\0\0\0"));
     info!("Reading {} sprites....", end_index - start_index);
     let t0 = time::precise_time_s();
-    for index in start_index .. end_index {
+    for index in start_index..end_index {
         let name = *wad.lump_name(index);
         match Image::from_buffer(&try!(wad.read_lump(index))) {
-            Ok(texture) => { textures.insert(name, texture); }
+            Ok(texture) => {
+                textures.insert(name, texture);
+            }
```

Comment:
Could you instead remove the braces and move on the same line as the `=>` please? It'll probably require a semicolon after the match... I think.